src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte

```diff
@@ -0,0 +1,471 @@
+            on:click={() => goToWorkflow(workflow)}
+          >
+            {#if $supportsBulkActions}
+              <td>
+                <Checkbox
+                  hoverable
+                  bind:group={selectedWorkflows}
+                  value={workflow}
+                  disabled={allSelected}
+                />
+              </td>
+            {/if}
+            {#each pinnedColumns as column}
+              <WorkflowsSummaryTableBodyCell {column} {workflow} />
+            {/each}
+          </tr>
+        {/each}
+      </tbody>
+    </table>
+  </div>
+  <div class="workflow-summary-table-wrapper">
+    <table class="workflow-summary-table">
+      <thead>
+        <tr class="workflow-summary-header-row">
+          {#each otherColumns as column}
+            <WorkflowsSummaryTableHeaderCell {column} {sortDisabled} />
+          {/each}
+          <th
+            class="px-2 h-10 w-12 text-right sticky right-0 flex items-center justify-center bg-primary"
+          >
+            <IconButton
+              icon="vertical-ellipsis"
+              on:click={openCustomizationDrawer}
+            />
+          </th>
+        </tr>
+        {#if $updating}
+          <ProgressBar />
+        {/if}
+      </thead>
+      <tbody>
+        {#each workflows as workflow}
+          <tr
+            class="workflow-summary-row"
+            on:click={() => goToWorkflow(workflow)}
+          >
+            {#each otherColumns as column}
+              <WorkflowsSummaryTableBodyCell {column} {workflow} />
+            {/each}
+            <td />
+          </tr>
+        {:else}
+          <tr>
+            <td colspan={otherColumns.length + 1}>
+              {#if $loading}
+                <Loading />
+              {:else}
+                <EmptyState
+                  title="No Workflows Found"
+                  content="If you have filters applied, try adjusting them. Otherwise please check your syntax and try again."
+                  error={$workflowError}
+                />
+              {/if}
+            </td>
+          </tr>
+        {/each}
+      </tbody>
+    </table>
+  </div>
+</div>
+
+<Drawer
+  open={customizationDrawerOpen}
+  onClick={closeCustomizationDrawer}
+  position="right"
+  dark={false}
+  title="Configure Workflow List"
+>
+  <svelte:fragment slot="subtitle">
+    <span>Add (</span><Icon class="inline" name="add" /><span
+      >), re-arrange (</span
+    ><Icon class="inline" name="chevron-selector-vertical" /><span
+      >), and remove (</span
+    ><Icon class="inline" name="hyphen" /><span
+      >), Workflow<br />Headings to personalize the Workflow List Table.</span
+    >
+  </svelte:fragment>
+  <div class="flex flex-col gap-4">
+    <OrderableList>
+      <svelte:fragment slot="heading">
+        Workflow Headings <span class="font-normal">(in view)</span>
+      </svelte:fragment>
+      {#each $workflowTableColumns as { label, pinned }, index (label)}
```

Comment:
Wondering if we want an empty state row here instead of it just becoming a collapsed border/line ⬇️ 

![Screenshot 2023-03-27 at 3 57 02 PM](https://user-images.githubusercontent.com/15069288/228078347-14aea9df-21e5-4268-9fce-7d7d5a0cc614.png)

Reply:
Good call, added an empty state in [bf62d26](https://github.com/temporalio/ui/pull/1229/commits/bf62d2663408730ce68ceb5055b418c4441dcadb)